mock/py/mockbuild/plugins/overlayfs.py

```diff
@@ -0,0 +1,815 @@
+# -*- coding: utf-8 -*-
+# This file is part of overlayfs plugin for mock
+# Copyright (C) 2018  Zdeněk Žamberský ( https://github.com/zzambers )
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
```

Comment:
Nowadays is preferred to use electronic address instead of the physical, which already changed few times. See "Copyright notice" at https://www.gnu.org/licenses/gpl-howto.html